mockserver-core/src/test/java/org/mockserver/persistence/ExpectationFileWatcherTest.java

```diff
@@ -308,7 +308,7 @@ public void shouldDetectModifiedInitialiserJsonOnAddWithGlob() throws Exception
             expectationFileWatcher = new ExpectationFileWatcher(configuration(), mockServerLogger, requestMatchers, new ExpectationInitializerLoader(configuration(), mockServerLogger, requestMatchers));
 
             // when
-            Expectation[] expections = {
+            Expectation[] expectations = {
```

Comment:
This will conflict with:
```java
            // then
            List<Expectation> expectations = requestMatchers.retrieveActiveExpectations(null);
```